### ISSUE TITLE
Total parameter shown in summary does not correlate with the number of layer in my model..I don't know if is the bug in Keras or it my model problem

### ISSUE THREAD

**Issue by georgekibirige** [keras-team/keras#12257]
This is the Model summary

Layer (type)                    Output Shape         Param #     Connected to                     
==================================================================================================
input_1 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_2 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_3 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_4 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_5 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_6 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_7 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_8 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_9 (InputLayer)            (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
input_10 (InputLayer)           (None, 1, 8, 1200, 1 0                                            
__________________________________________________________________________________________________
average_pooling3d_1 (AveragePoo (None, 1, 8, 240, 24 0           input_1[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_4 (AveragePoo (None, 1, 8, 240, 24 0           input_2[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_7 (AveragePoo (None, 1, 8, 240, 24 0           input_3[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_10 (AveragePo (None, 1, 8, 240, 24 0           input_4[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_13 (AveragePo (None, 1, 8, 240, 24 0           input_5[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_16 (AveragePo (None, 1, 8, 240, 24 0           input_6[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_19 (AveragePo (None, 1, 8, 240, 24 0           input_7[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_22 (AveragePo (None, 1, 8, 240, 24 0           input_8[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_25 (AveragePo (None, 1, 8, 240, 24 0           input_9[0][0]                    
__________________________________________________________________________________________________
average_pooling3d_28 (AveragePo (None, 1, 8, 240, 24 0           input_10[0][0]                   
__________________________________________________________________________________________________
reshape_1 (Reshape)             (None, 8, 1, 240, 24 0           average_pooling3d_1[0][0]        
__________________________________________________________________________________________________
reshape_4 (Reshape)             (None, 8, 1, 240, 24 0           average_pooling3d_4[0][0]        
__________________________________________________________________________________________________
reshape_7 (Reshape)             (None, 8, 1, 240, 24 0           average_pooling3d_7[0][0]        
__________________________________________________________________________________________________
reshape_10 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_10[0][0]       
__________________________________________________________________________________________________
reshape_13 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_13[0][0]       
__________________________________________________________________________________________________
reshape_16 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_16[0][0]       
__________________________________________________________________________________________________
reshape_19 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_19[0][0]       
__________________________________________________________________________________________________
reshape_22 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_22[0][0]       
__________________________________________________________________________________________________
reshape_25 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_25[0][0]       
__________________________________________________________________________________________________
reshape_28 (Reshape)            (None, 8, 1, 240, 24 0           average_pooling3d_28[0][0]       
__________________________________________________________________________________________________
conv_lst_m2d_1 (ConvLSTM2D)     (None, 8, 16, 240, 2 9856        reshape_1[0][0]                  
__________________________________________________________________________________________________
conv_lst_m2d_3 (ConvLSTM2D)     (None, 8, 16, 240, 2 9856        reshape_4[0][0]                  
__________________________________________________________________________________________________
conv_lst_m2d_5 (ConvLSTM2D)     (None, 8, 16, 240, 2 9856        reshape_7[0][0]                  
__________________________________________________________________________________________________
conv_lst_m2d_7 (ConvLSTM2D)     (None, 8, 16, 240, 2 9856        reshape_10[0][0]                 
__________________________________________________________________________________________________
conv_lst_m2d_9 (ConvLSTM2D)     (None, 8, 16, 240, 2 9856        reshape_13[0][0]                 
__________________________________________________________________________________________________
conv_lst_m2d_11 (ConvLSTM2D)    (None, 8, 16, 240, 2 9856        reshape_16[0][0]                 
__________________________________________________________________________________________________
conv_lst_m2d_13 (ConvLSTM2D)    (None, 8, 16, 240, 2 9856        reshape_19[0][0]                 
__________________________________________________________________________________________________
conv_lst_m2d_15 (ConvLSTM2D)    (None, 8, 16, 240, 2 9856        reshape_22[0][0]                 
__________________________________________________________________________________________________
conv_lst_m2d_17 (ConvLSTM2D)    (None, 8, 16, 240, 2 9856        reshape_25[0][0]                 
__________________________________________________________________________________________________
conv_lst_m2d_19 (ConvLSTM2D)    (None, 8, 16, 240, 2 9856        reshape_28[0][0]                 
__________________________________________________________________________________________________
batch_normalization_1 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_1[0][0]             
__________________________________________________________________________________________________
batch_normalization_2 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_3[0][0]             
__________________________________________________________________________________________________
batch_normalization_3 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_5[0][0]             
__________________________________________________________________________________________________
batch_normalization_4 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_7[0][0]             
__________________________________________________________________________________________________
batch_normalization_5 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_9[0][0]             
__________________________________________________________________________________________________
batch_normalization_6 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_11[0][0]            
__________________________________________________________________________________________________
batch_normalization_7 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_13[0][0]            
__________________________________________________________________________________________________
batch_normalization_8 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_15[0][0]            
__________________________________________________________________________________________________
batch_normalization_9 (BatchNor (None, 8, 16, 240, 2 960         conv_lst_m2d_17[0][0]            
__________________________________________________________________________________________________
batch_normalization_10 (BatchNo (None, 8, 16, 240, 2 960         conv_lst_m2d_19[0][0]            
__________________________________________________________________________________________________
conv_lst_m2d_2 (ConvLSTM2D)     (None, 8, 16, 240, 2 18496       batch_normalization_1[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_4 (ConvLSTM2D)     (None, 8, 16, 240, 2 18496       batch_normalization_2[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_6 (ConvLSTM2D)     (None, 8, 16, 240, 2 18496       batch_normalization_3[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_8 (ConvLSTM2D)     (None, 8, 16, 240, 2 18496       batch_normalization_4[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_10 (ConvLSTM2D)    (None, 8, 16, 240, 2 18496       batch_normalization_5[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_12 (ConvLSTM2D)    (None, 8, 16, 240, 2 18496       batch_normalization_6[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_14 (ConvLSTM2D)    (None, 8, 16, 240, 2 18496       batch_normalization_7[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_16 (ConvLSTM2D)    (None, 8, 16, 240, 2 18496       batch_normalization_8[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_18 (ConvLSTM2D)    (None, 8, 16, 240, 2 18496       batch_normalization_9[0][0]      
__________________________________________________________________________________________________
conv_lst_m2d_20 (ConvLSTM2D)    (None, 8, 16, 240, 2 18496       batch_normalization_10[0][0]     
__________________________________________________________________________________________________
reshape_2 (Reshape)             (None, 16, 8, 240, 2 0           conv_lst_m2d_2[0][0]             
__________________________________________________________________________________________________
reshape_5 (Reshape)             (None, 16, 8, 240, 2 0           conv_lst_m2d_4[0][0]             
__________________________________________________________________________________________________
reshape_8 (Reshape)             (None, 16, 8, 240, 2 0           conv_lst_m2d_6[0][0]             
__________________________________________________________________________________________________
reshape_11 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_8[0][0]             
__________________________________________________________________________________________________
reshape_14 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_10[0][0]            
__________________________________________________________________________________________________
reshape_17 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_12[0][0]            
__________________________________________________________________________________________________
reshape_20 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_14[0][0]            
__________________________________________________________________________________________________
reshape_23 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_16[0][0]            
__________________________________________________________________________________________________
reshape_26 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_18[0][0]            
__________________________________________________________________________________________________
reshape_29 (Reshape)            (None, 16, 8, 240, 2 0           conv_lst_m2d_20[0][0]            
__________________________________________________________________________________________________
average_pooling3d_2 (AveragePoo (None, 16, 8, 80, 80 0           reshape_2[0][0]                  
__________________________________________________________________________________________________
average_pooling3d_5 (AveragePoo (None, 16, 8, 80, 80 0           reshape_5[0][0]                  
__________________________________________________________________________________________________
average_pooling3d_8 (AveragePoo (None, 16, 8, 80, 80 0           reshape_8[0][0]                  
__________________________________________________________________________________________________
average_pooling3d_11 (AveragePo (None, 16, 8, 80, 80 0           reshape_11[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_14 (AveragePo (None, 16, 8, 80, 80 0           reshape_14[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_17 (AveragePo (None, 16, 8, 80, 80 0           reshape_17[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_20 (AveragePo (None, 16, 8, 80, 80 0           reshape_20[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_23 (AveragePo (None, 16, 8, 80, 80 0           reshape_23[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_26 (AveragePo (None, 16, 8, 80, 80 0           reshape_26[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_29 (AveragePo (None, 16, 8, 80, 80 0           reshape_29[0][0]                 
__________________________________________________________________________________________________
average_pooling3d_3 (AveragePoo (None, 16, 8, 8, 8)  0           average_pooling3d_2[0][0]        
__________________________________________________________________________________________________
average_pooling3d_6 (AveragePoo (None, 16, 8, 8, 8)  0           average_pooling3d_5[0][0]        
__________________________________________________________________________________________________
average_pooling3d_9 (AveragePoo (None, 16, 8, 8, 8)  0           average_pooling3d_8[0][0]        
__________________________________________________________________________________________________
average_pooling3d_12 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_11[0][0]       
__________________________________________________________________________________________________
average_pooling3d_15 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_14[0][0]       
__________________________________________________________________________________________________
average_pooling3d_18 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_17[0][0]       
__________________________________________________________________________________________________
average_pooling3d_21 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_20[0][0]       
__________________________________________________________________________________________________
average_pooling3d_24 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_23[0][0]       
__________________________________________________________________________________________________
average_pooling3d_27 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_26[0][0]       
__________________________________________________________________________________________________
average_pooling3d_30 (AveragePo (None, 16, 8, 8, 8)  0           average_pooling3d_29[0][0]       
__________________________________________________________________________________________________
reshape_3 (Reshape)             (None, 8, 16, 8, 8)  0           average_pooling3d_3[0][0]        
__________________________________________________________________________________________________
reshape_6 (Reshape)             (None, 8, 16, 8, 8)  0           average_pooling3d_6[0][0]        
__________________________________________________________________________________________________
reshape_9 (Reshape)             (None, 8, 16, 8, 8)  0           average_pooling3d_9[0][0]        
__________________________________________________________________________________________________
reshape_12 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_12[0][0]       
__________________________________________________________________________________________________
reshape_15 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_15[0][0]       
__________________________________________________________________________________________________
reshape_18 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_18[0][0]       
__________________________________________________________________________________________________
reshape_21 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_21[0][0]       
__________________________________________________________________________________________________
reshape_24 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_24[0][0]       
__________________________________________________________________________________________________
reshape_27 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_27[0][0]       
__________________________________________________________________________________________________
reshape_30 (Reshape)            (None, 8, 16, 8, 8)  0           average_pooling3d_30[0][0]       
__________________________________________________________________________________________________
concatenate_1 (Concatenate)     (None, 8, 16, 8, 80) 0           reshape_3[0][0]                  
                                                                 reshape_6[0][0]                  
                                                                 reshape_9[0][0]                  
                                                                 reshape_12[0][0]                 
                                                                 reshape_15[0][0]                 
                                                                 reshape_18[0][0]                 
                                                                 reshape_21[0][0]                 
                                                                 reshape_24[0][0]                 
                                                                 reshape_27[0][0]                 
                                                                 reshape_30[0][0]                 
__________________________________________________________________________________________________
reshape_31 (Reshape)            (None, 8, 10240)     0           concatenate_1[0][0]              
__________________________________________________________________________________________________
time_distributed_1 (TimeDistrib (None, 8, 8)         81928       reshape_31[0][0]                 
__________________________________________________________________________________________________
dropout_1 (Dropout)             (None, 8, 8)         0           time_distributed_1[0][0]         
__________________________________________________________________________________________________
time_distributed_2 (TimeDistrib (None, 8, 8)         72          dropout_1[0][0]                  
__________________________________________________________________________________________________
dropout_2 (Dropout)             (None, 8, 8)         0           time_distributed_2[0][0]         
__________________________________________________________________________________________________
time_distributed_3 (TimeDistrib (None, 8, 2)         18          dropout_2[0][0]                  
==================================================================================================
Total params: 375,138
Trainable params: 370,338
Non-trainable params: 4,800
__________________________________________________________________________________________________

Thank you!

**Comment by georgekibirige** [comment]
That model has ten ConvLSTM layers and three dense layers, Later i merge the output of each ConvLSTM layers and their output go through those dense layer. SO how come Total parameters shown in summary is very small
